src/lib/Form.svelte

```diff
@@ -21,33 +15,56 @@
   }
 </script>
 
-<TextInput labelText="Name" bind:value={name} />
+<label
```

Comment:
https://stackoverflow.com/questions/774054/should-i-put-input-elements-inside-a-label-element
No need to repeat IDs and `label for`